packages/plugin-ext/src/plugin/plugin-manager.ts

```diff
@@ -266,6 +267,7 @@ export class PluginManagerExtImpl implements PluginManagerExt, PluginManager {
         const asAbsolutePath = (relativePath: string): string => join(plugin.pluginFolder, relativePath);
         const logPath = join(configStorage.hostLogPath, plugin.model.id); // todo check format
         const storagePath = join(configStorage.hostStoragePath || '', plugin.model.id);
+        const globalStoragePath = join(configStorage.hostGlobalStoragePath || os.homedir(), plugin.model.id);
```

Comment:
`~/${plugin.model.id}` it is now; should we move it to `~/.theia/globalStorage`?